src/ssl.c

```diff
@@ -19121,14 +19272,27 @@ int wolfSSL_ASN1_STRING_to_UTF8(unsigned char **out, WOLFSSL_ASN1_STRING *in)
        The length of out is returned or a negative error code.
        The buffer *out should be free using OPENSSL_free().
        */
-    (void)out;
-    (void)in;
-    WOLFSSL_STUB("ASN1_STRING_to_UTF8");
-    return -1;
+    unsigned char* buf;
+    int inLen;
+
+    if (!out || !in) {
+        return -1;
+    }
+
+    inLen = wolfSSL_ASN1_STRING_length(in);
```

Comment:
right now wolfSSL_ASN1_STRING_length always returns 0 or greater. I think it is slightly dangerous to always assume the int returned in the future will be though.